### ISSUE TITLE
? provide for consuming/producing yaml

### ISSUE THREAD

**Issue by geemus** [interagent/prmd#16]
I saw another case (related to chef) where somebody allowed you to provide yaml/json interchangeably.  I cringe a little whenever I have to deal with significant whitespace, but it is hard to deny that yaml is easier (and fewer keystrokes) for creating/editing by humans. As a superset of json it also should be pretty straightforward to convert when building the full schema. This might help mitigate some of the desire for a more DSLish approach with a minimal amount of effort involved.

**Comment by waterlink** [comment]
Hi @geemus, I have found this feature to be awesome, since I tried to make a conversion myself, and then discovered that it already recognises my YAML files.

So my questions are following: Can I rely on this feature? Is it documented somewhere? Will not it be removed in one of the following minor/patch releases?

**Comment by geemus** [comment]
@waterlink hey. I don't think we documented it very well, so certainly would welcome help there. In our usage we rely heavily on YAML. At first we just added it to try it out and weren't sure if it would be a good idea or we would like it (which I think is how we ended up not documenting it at the time). Now I can't really imagine going back (YAML is much easier to edit/type since you don't have to use nearly as many opening/closing marks). So I think it should be safe to say it will be around.
